package.json

```diff
@@ -55,6 +55,7 @@
     "@ngrx/store": "^11.1.1",
     "@ngx-dino/core": "^0.11.0",
     "@ngx-dino/legend": "^0.13.0",
+    "@ngxs/store": "^3.7.2",
```

Comment:
Is @ngxs/store needed?

Reply:
No I don't think it's being used anywhere. Should I remove it?

Reply:
Yes please